packages/framework-provider-azure-infrastructure/src/infrastructure/index.ts

```diff
@@ -36,7 +42,13 @@ async function deployApp(config: BoosterConfig, rockets?: InfrastructureRocket[]
     return Promise.reject(`Deployment of application ${config.appName} failed. Check cdktf logs. \n${error.message}}`)
   }
 
-  await applicationBuilder.uploadFile(applicationBuild.zipResource)
+  const resourceGroupName = createResourceGroupName(config.appName, config.environmentName)
+  const functionAppName = createFunctionResourceGroupName(resourceGroupName)
+  await FunctionZip.deployZip(config, functionAppName, resourceGroupName, applicationBuild.zipResource)
+  if (config.eventStreamConfiguration.enabled) {
+    const streamFunctionAppName = createStreamFunctionResourceGroupName(resourceGroupName)
+    await FunctionZip.deployZip(config, streamFunctionAppName, resourceGroupName, applicationBuild.consumerZipResource)
+  }
```

Comment:
If I understood correctly, now we have two different zips, although the code is the same... Is this true?
If so, Can't we use the same zip resource as we do for all the other functions?

Reply:
It's not exactly the same as the definitions of the functions in each file are different. The `zipResource` and the `consumerZipResource` are differents:

```
    const zipResource = await FunctionZip.copyZip(azureStack.applicationStack.functionDefinitions!, 'functionApp.zip')
    const consumerZipResource = await FunctionZip.copyZip(
      azureStack.applicationStack.consumerFunctionDefinitions!,
      'consumerFunctionApp.zip'
    )
```